### ISSUE TITLE
Improve IntegratePeakProfileFitting suitability for WISH

### ISSUE THREAD

**Issue by martyngigg** [mantidproject/mantid#26488]
`IntegratePeaksProfileFitting` currently accepts some parameters, pulls some from the instrument and has 2 input files: `ModeratorCoefficientsFile` & `StrongPeakParamsFile`. 

WISH would like to specify the moderator coefficients derived from real data and not simulations. Investigate the file format of both of the above files and if they are simple then provide a script to create them other wise create another issue to have better inputs for inputting real data here.

**Comment by DannyHindson** [comment]
Waiting on input\readiness from WISH
